commons/commands.py

```diff
@@ -131,7 +130,7 @@
 SEL_LIST_CMD = "ipmitool sel list"
 IEM_LOGGER_CMD = "logger -i -p local3.err {}"
 INSTALL_SCREEN_CMD = "yum -y install screen"
-INSTALL_SSH_PASS_CMD = "yum -y install sshpass"
+INSTALL_SSH_PASS_CMD = "yum -y install sshpass"  # nosec
```

Comment:
It's not a false alarm and bandit needs to fix the name of the variable itself. It does not like the words like PASS, PWD, PASSWORD, etc.,

Reply:
We can refactor this later, added this for command.